examples/kubernetes-example-annotationless-config/readme.md

```diff
@@ -0,0 +1,92 @@
+# Add configmap data to a volume Example 
+
+An example that showcase how to generate a kubernetes resource utilizing Spring Boot-specific metadata. 
+To customize the generated manifests you need to add dekorate properties to your `application.yml` or `application.properties` descriptors. 
+and have the following dependency in your class path:
+
+    <dependency>
+          <groupId>io.dekorate</groupId>
+          <artifactId>dekorate-spring-boot</artifactId>
+          <version>${project.version}</version>
+    </dependency>
+
+Compile the project using:
+
+    mvn clean install
+    
+You can find the generated deployment under: `target/classes/dekorate/kubernetes.yml` that should look like:
+```---
+   apiVersion: "v1"
+   kind: "List"
```

Comment:
Don't list all the resources generated but only the section which is relevant ... Deployment in this case ?

Reply:
done